ooxml/POIXMLDocument.cs

```diff
@@ -68,11 +68,11 @@ private void init(OPCPackage pkg)
          *  in the event of a problem.
          * Works around shortcomings in java's this() constructor calls
          */
-        public static OPCPackage OpenPackage(String path)
+        public static OPCPackage OpenPackage(String path, bool bReadOnly=false)
```

Comment:
would just `readOnly` be more intuitive without the type qualifier in front?  this applies to all places in PR

also this is missing spaces between the equals sign